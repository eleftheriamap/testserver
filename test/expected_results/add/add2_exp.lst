
test/exp/test/gen/add/add2_exp.bin:     file format binary


Disassembly of section .data:

0000000000000000 <.data>:
   0:	5126794a 	sub	w10, w10, #0x99e
   4:	5171f4c6 	sub	w6, w6, #0xc7d, lsl #12
   8:	0b060145 	add	w5, w10, w6
   c:	8a000000 	and	x0, x0, x0
