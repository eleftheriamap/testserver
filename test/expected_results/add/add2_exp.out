Registers:
X00    = 0000000000000000
X01    = 0000000000000000
X02    = 0000000000000000
X03    = 0000000000000000
X04    = 0000000000000000
X05    = 00000000ff382662
X06    = 00000000ff383000
X07    = 0000000000000000
X08    = 0000000000000000
X09    = 0000000000000000
X10    = 00000000fffff662
X11    = 0000000000000000
X12    = 0000000000000000
X13    = 0000000000000000
X14    = 0000000000000000
X15    = 0000000000000000
X16    = 0000000000000000
X17    = 0000000000000000
X18    = 0000000000000000
X19    = 0000000000000000
X20    = 0000000000000000
X21    = 0000000000000000
X22    = 0000000000000000
X23    = 0000000000000000
X24    = 0000000000000000
X25    = 0000000000000000
X26    = 0000000000000000
X27    = 0000000000000000
X28    = 0000000000000000
X29    = 0000000000000000
X30    = 0000000000000000
PC     = 000000000000000c
PSTATE : -Z--
Non-zero memory:
0x00000000 : 0x5126794a
0x00000004 : 0x5171f4c6
0x00000008 : 0x0b060145
0x0000000c : 0x8a000000
